aws_lambda_powertools/utilities/data_classes/shared_functions.py

```diff
@@ -84,3 +84,28 @@ def get_query_string_value(
     """
     params = query_string_parameters
     return default_value if params is None else params.get(name, default_value)
+
+
+def get_multi_value_query_string_values(
+    multi_value_query_string_parameters: Dict[str, list[str]] | None,
+    name: str,
+    default_values: list[str] | None = None,
+) -> list[str] | None:
+    """
+    Retrieves the values of a multi-value string parameters specified by the given name.
+
+    Parameters
+    ----------
+    name: str
+        The name of the query string parameter to retrieve.
+    default_value: list[str], optional
+        The default value to return if the parameter is not found. Defaults to None.
+
+    Returns
+    -------
+    List[str]. optional
+        The values of the query string parameter if found, or the default values if not found.
+    """
+
+    params = multi_value_query_string_parameters
+    return default_values if not params else params.get(name, default_values)
```

Comment:
```suggestion
    default = default_values or []
    params = multi_value_query_string_parameters or {}

    return params.get(name) or default
```

defensively create defaults so we can always return a list, making it easier for whoever uses it.